store/tikv/region_request.go

```diff
@@ -15,6 +15,7 @@ package tikv
 
 import (
 	"context"
+	"github.com/pingcap/tidb/config"
```

Comment:
The `github.com/**` should be placed in an individual group.

Reply:
Addressed,Done.